packages/data-stores/src/wpcom-request-controls/index.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import wpcomProxyRequest from 'wpcom-proxy-request';
+import debugFactory from 'debug';
+
+type WpcomProxyRequestOptions = Parameters< typeof wpcomProxyRequest >[ 0 ];
+
+const debug = debugFactory( 'data-stores:utils:wpcom-wrapper' );
+
+export const wpcomRequest = ( request: WpcomProxyRequestOptions ) =>
+	( { type: 'WPCOM_REQUEST', request } as const );
+
+export interface WpcomClientCredentials {
+	client_id: string;
+	client_secret: string;
+}
```

Comment:
Doesn't belong here, this seems quite specific to signup or `/me` endpoint.

Reply:
Good point, it makes sense for this to belong somewhere over in the User store.

Reply:
I was confused because I thought these were necessary for wpcom-proxy-request, but they're just used by some of the signup endpoints if I've understood correctly.

Reply:
I've moved this to `shared-types.ts`